app/containers/Applications/ApplicationWizard.tsx

```diff
@@ -42,6 +43,11 @@ export const ApplicationWizardComponent: React.FunctionComponent<Props> = ({
     latestSubmittedRevision,
     applicationRevisionByStringVersionNumber: applicationRevision
   } = application;
+  const reviewComments = query.application.reviewCommentsByApplicationId?.edges.map(
+    ({node: description}) => description
```

Comment:
```suggestion
    ({node: {description}}) => description
```
what you wrote above renames `node` to `description`